saskatoon/member/models.py

```diff
@@ -113,10 +113,22 @@ def get_person(self):
     def get_organization(self):
         return Organization.objects.filter(actor_id=self.actor_id).first()
 
-    def __str__(self):
+    @property
+    def is_person(self):
         if self.get_person():
+            return True
+        return False
+
+    @property
+    def is_organization(self):
+        if self.get_organization():
+            return True
+        return False
```

Comment:
@victorphoenix3 I'm not sure if using the `get_person()` / `get_organisation()` methods (i.e. filter through all `Person` / `Organization` ) is the most straightforward. What I had initially in mind was:
`return hasattr(self, 'person')` / `return hasattr(self, 'organization'). What do you think?
If not would something like `return self.get_person() is not None` work?